@navikt/internal/css/timeline.css

```diff
@@ -0,0 +1,222 @@
+  all: unset;
+}
+
+.navdsi-timeline__period {
+  height: calc(100% - 2px);
+  border-radius: 20px;
+  position: absolute;
+  border: 1px solid;
+  padding: 0;
+  display: flex;
+  align-items: center;
+  z-index: 2;
+}
+
+.navdsi-timeline__period > * {
+  padding-left: var(--navds-spacing-2);
+}
+
+.navdsi-timeline__period--success {
+  background: var(--navds-global-color-green-50);
+  border-color: var(--navds-global-color-green-500);
+}
+
+.navdsi-timeline__period--success[data-clickable]:hover {
+  background: var(--navds-global-color-green-100);
+}
+
+.navdsi-timeline__period--warning {
+  background: var(--navds-global-color-orange-50);
+  border-color: var(--navds-global-color-orange-600);
+}
+
+.navdsi-timeline__period--warning[data-clickable]:hover {
+  background: var(--navds-global-color-orange-100);
+}
+
+.navdsi-timeline__period--danger {
+  background: var(--navds-global-color-red-50);
+  border-color: var(--navds-global-color-red-500);
+}
+
+.navdsi-timeline__period--danger[data-clickable]:hover {
+  background: var(--navds-global-color-red-100);
+}
+
+.navdsi-timeline__period--information {
+  background: var(--navds-global-color-lightblue-50);
+  border-color: var(--navds-global-color-lightblue-700);
+}
+
+.navdsi-timeline__period--information[data-clickable]:hover {
+  background: var(--navds-global-color-lightblue-100);
+}
+
+.navdsi-timeline__period--default {
+  background: var(--navds-global-color-gray-50);
+  border-color: var(--navds-global-color-gray-600);
+}
+
+.navdsi-timeline__period--default[data-clickable]:hover {
+  background: var(--navds-global-color-gray-100);
+}
+
+.navdsi-timeline__period[data-clickable]:hover {
+  cursor: pointer;
+}
+
+.navdsi-timeline__period--connectedBoth {
+  border-radius: 0;
+}
+
+.navdsi-timeline__period--connectedRight {
+  border-bottom-right-radius: 0;
+  border-top-right-radius: 0;
+}
+
+.navdsi-timeline__period--connectedLeft {
+  border-bottom-left-radius: 0;
+  border-top-left-radius: 0;
+}
+
+.navdsi-timeline__period--selected {
+  box-shadow: 0 0 0 1px var(--navds-global-color-deepblue-500);
+  border-color: var(--navds-global-color-deepblue-500);
+  height: calc(100% - 4px);
+  transform: translateY(1px);
+  margin-right: -2px;
+  z-index: 3;
+}
+
+.navdsi-timeline__pin__wrapper {
+  position: absolute;
+  grid-column: 2;
+  top: 0;
+  z-index: 1;
+  height: 100%;
+  display: flex;
+  flex-direction: column-reverse;
+}
+
+.navdsi-timeline__pin__button {
+  cursor: pointer;
+  background: #fff;
+  border: 3px solid var(--navds-global-color-red-500);
+  z-index: 2;
+  border-radius: var(--navds-border-radius-full);
+  padding: 2.5px;
+}
+
+.navdsi-timeline__pin__wrapper::before {
+  content: " ";
+  top: 13px;
+  height: calc(100% - 13px);
+  width: 1px;
+  margin: 0 auto;
+  background: var(--navds-global-color-gray-900);
+}
+
+.navdsi-timeline__zoom {
+  margin-top: var(--navds-spacing-6);
+  float: right;
+}
+
+.navdsi-timeline__zoom__button {
```

Comment:
Er litt problemer med border på fokus, kan se i toggle-group.css for hvordan det bl fikset der (tror det var box-shadow)